cfscrape/__init__.py

```diff
@@ -10,6 +10,7 @@
 import os
 from base64 import b64encode
 from collections import OrderedDict
+from lxml import etree
```

Comment:
I am a little wary of introducing this new dependency for this, not all systems have libxml2 installed (required for lxml). I am sure that for the methods we are using below, regex should suffice. I know that the container I am running this in, does not - and if it gets introduced will require a rebuild. Not difficult, just a pain. Open to a discussion on it.

Reply:
hmm, i have no problem with regrex and can rewirte it to be only regrex code (need to make code so it has pattern that find best solution)
it just i'm windows user so i dont know these problem that come with non-windows user, that why i did not know that libxml2 is system dependency and always thought it come with Lxml package

Reply:
Not a problem! Totally understandable, that is why there are public reviews! So others can give additional input. The code you had before could still work, the regex might just need to be changed slightly. If you need help with the regex, or the pattern finding, feel free to reach out :)

Reply:
ok i re-updated the code with only regrex
i used multi step search in regrex to insure that i get the right data
for example:  i found challenge form first then let it find data to input
why did i do that?
its simple
because as i said at lxml regrex is good to find data but not flexable
as for html <input type="hidden" name="jschl_vc" value="c22565a73c49afb92b427eff230eb330"/> and <input type="hidden"  value="c22565a73c49afb92b427eff230eb330" name="jschl_vc"/> is the same (the order was changed) but regrex dont  find them like that
that why i split the searching and did it as phase:
find challenge form > find input name> find input value> check the method> then add it to the right type of data (url params or data)

Reply:
Introducing lxml as a dependency can wreck havoc on people running this on an Alpine dockers, please try to avoid that.

Reply:
This has already been addressed in the latest push :)

Reply:
@098799
hmm i already re-write the code using only regrex and removed all lxml

Reply:
@098799 Off topic, but what issues have you seen using lxml in Alpine containers?  I've been doing that for quite some time without trouble

Reply:
@mcronce You need `libxml2`, `libxml2-dev` and `libxslt-dev` on top of `lxml`. Just rebuilt the images to check the their sizes and it seems like this addition changes the size of the docker image from ~100 MB to 184 MB.

Reply:
lol i already re-wrote the code and removed lxml